lesson20/homework.py

```diff
@@ -0,0 +1,51 @@
+from pathlib import Path
+import logging
+from datetime import datetime
+
+log_file_path = Path(__file__).parent / "hb_test.log"
+log_file_path.parent.mkdir(parents=True, exist_ok=True)
+
+logging.basicConfig(filename=str(log_file_path), level=logging.DEBUG,
+                    format='%(asctime)s - %(levelname)s - %(message)s')
+
+def time_to_seconds(time_str):
+    try:
+        time_obj = datetime.strptime(time_str, "%H:%M:%S")
+        return time_obj.hour * 3600 + time_obj.minute * 60 + time_obj.second
```

Comment:
```suggestion
        return time_obj
```
а чому б просто не повертати об'єкт часу?